aten/src/ATen/native/native_functions.yaml

```diff
@@ -3915,28 +3909,28 @@
 
 - func: lt.Scalar_out(Tensor self, Scalar other, *, Tensor(a!) out) -> Tensor(a!)
   dispatch:
-    CPU: lt_scalar_out_cpu
-    CUDA: lt_scalar_out_cuda
+    CPU: lt_out
+    CUDA: lt_out
     QuantizedCPU: lt_out_quantized_cpu
 
 - func: lt.Scalar(Tensor self, Scalar other) -> Tensor
   variants: method, function
   dispatch:
-    CPU: legacy::cpu::_th_lt
-    CUDA: legacy::cuda::_th_lt
+    CPU: lt
+    CUDA: lt
```

Comment:
can we exclude these CPU/CUDA lines when the function name matches?

Reply:
We can't because QuantizedCPU has its own dispatch option so we need to specify all dispatch paths.

Reply:
we probably also want to delete the implemented TH functions in THTensorMath etc

Reply:
Old implementation has been removed